docs/running-on-mesos.md

```diff
@@ -387,6 +387,13 @@ See the [configuration page](configuration.html) for information on Spark config
     </ul>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.coarse.shutdown.ms</code></td>
+  <td><code>10000</code> (10 seconds)</td>
+  <td> 
+    Time (in ms) to wait for executors to report that they have exited. Setting this too low has the risk of shutting down the Mesos driver (and thereby killing the spark executors) while the executor is still in the process of exiting cleanly.
```

Comment:
ah I see. @dragos do you see any potential use case for users to actually set this? I would think the user always wants to wait until his temp files are cleaned up but doesn't want to block forever. The default value of 10s probably satisfies this 99% of the time.